biofefi/options/file_paths.py

```diff
@@ -187,6 +201,26 @@ def execution_options_path(experiment_path: Path) -> Path:
     return experiment_path / "execution_options.json"
 
 
+def data_preprocessing_options_path(experiment_path: Path) -> Path:
```

Comment:
Like, `raw_data_path` can you add a test in `tests/options/test_file_paths.py` please